plugins/inputs/amqp_consumer/amqp_consumer.go

```diff
@@ -62,11 +64,12 @@ type AMQPConsumer struct {
 	decoder internal.ContentDecoder
 }
 
-type externalAuth struct{}
-
+// Mechanism returns "EXTERNAL"
 func (a *externalAuth) Mechanism() string {
 	return "EXTERNAL"
 }
+
+// Response returns "\000"
 func (a *externalAuth) Response() string {
 	return "\000"
 }
```

Comment:
Please revert this! Keep the struct close to the method definition of that struct and remove those comments telling you exactly the same as the code does!

Reply:
Changed